packages/stream_chat_flutter/lib/src/message_input.dart

```diff
@@ -148,11 +148,16 @@ class MessageInput extends StatefulWidget {
     this.activeSendButton,
     this.showCommandsButton = true,
     this.mentionsTileBuilder,
+    this.maxAttachmentSize = _kDefaultMaxAttachmentSize,
   }) : super(key: key);
 
   /// Message to edit
   final Message? editMessage;
 
+  /// Max attachment size in bytes
+  /// If you're using our default CDN do not set it
```

Comment:
maybe something like
`defaults to 20MB (Stream CDN)`
@deven98 can suggest something better